### PR TITLE
Expose `relation` and `uriTemplate` for actions

### PR DESCRIPTION
**Pull request by klokane**

### CODE REVIEW

File: src/legacy/blueprint.coffee [hunk 1]
```diff
@@ -218,6 +218,8 @@ class Resource
       resolvedAttributes:  undefined
       actionAttributes:    undefined
       resolvedActionAttributes: undefined
+      actionRelation:      undefined
```
Comment (zdne) [inline src/legacy/blueprint.coffee:223]: @klokane you need to add this to `toJSON` and `fromJSON` in order for serialization to work, pity it is not tested ...

**Comment by zdne** [comment]
@klokane looks good except missing serialization and deserialization for the introduced attributes, please add it and it is good to merge

File: src/legacy/blueprint.coffee (hunk 1 above)
Comment (klokane) [inline src/legacy/blueprint.coffee:223]: How do i invoke it in tests?

**Comment by klokane** [comment]
@zdne done

**Pull request merged**